src/bitmessagekivy/uikivysignaler.py

```diff
@@ -0,0 +1,34 @@
+"""
+    UI Singnaler for kivy interface
+"""
+
+from threading import Thread
+from kivy.app import App
+import queues
+import state
+
+from debug import logger
+from bitmessagekivy.baseclass.common import kivy_state_variables
+
+
+class UIkivySignaler(Thread):
+    """Kivy ui signaler"""
+
+    def __init__(self, *args, **kwargs):
+        super(UIkivySignaler, self).__init__(*args, **kwargs)
+        self.kivy_state = kivy_state_variables()
+
+    def run(self):
+        self.kivy_state.sql_ready.wait()
```

Comment:
don't use `sql_ready`, create a new `threading.Event` object, just use a different name. Just look at how `sql_ready` is used so that you understand what to do.